slm_porosity/heat_transfer.f90

```diff
@@ -458,8 +455,22 @@ SUBROUTINE user_stats ( u ,j_mn, startup_flag)
     INTEGER , INTENT (IN) :: startup_flag
     INTEGER , INTENT (IN) :: j_mn
     REAL (pr), DIMENSION (nwlt,1:n_var), INTENT (IN) :: u
-  END SUBROUTINE user_stats
+    REAL (pr) :: liquid_sum
+    REAL(pr), DIMENSION(1) :: width, length, depth
 
+    IF (startup_flag.EQ.0) THEN
+      OPEN(12, file="liquid_fraction.txt", status="replace", action="write")
+      CLOSE(12)
+    ELSE
+      liquid_sum = SUM(u(:,n_var_lfrac)*dA(:))
```

Comment:
What is the `liquid_sum`?! Let it be `volume`, but add some clarifying comment for all quantities, like `Dimensions of the melting pool`.